src/test/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -0,0 +1 @@
+org.ice4j.TestExecutionListener
```

Comment:
What is this?

Reply:
Activates the test listener to stop the caching
https://junit.org/junit5/docs/current/user-guide/#launcher-api-listeners-custom